example/index.html

```diff
@@ -8,70 +8,108 @@
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 <meta name="robots" content="noindex, nofollow" />
 <style>
-	button {
-			width: 100px;
-			height: 44px;
-	}
+    button {
+        width: 120px;
+        height: 44px;
+    }
 </style>
 
 </head>
 <body>
   <h1>Notify.js demo</h1>
   <p>Notify.js is a simple wrapper for the Web Notification API. The GitHub repo is <a href="https://github.com/alexgibson/notify.js">here</a>.</p>
-  <p>Click the button below to fire up a notification. Currently supported in Chrome, Safari, Firefox and Firefox OS.</p>
-  <button id="my-button">Notify Me!</button>
+  <p>Click the buttons below to fire up a notification. Currently supported in Chrome, Safari, Firefox and Firefox OS.</p>
+  <button id="btn-api">Notify Me with API!</button>
+  <button id="btn-service-worker">Notify Me with Service Worker!</button>
```

Comment:
I think this use case might be a bit confusing, and possible not needed. I think by example we should be demonstrating notifications triggered from a SW only, since this is the only way push will work.